pkg/deployer/deployer.go

```diff
@@ -23,6 +23,19 @@ import (
 	"sigs.k8s.io/cluster-api-provider-vsphere/pkg/cloud/vsphere/services/kubeclient"
 )
 
+var d Deployer
+
+// GetIP returns the control plane endpoint for the cluster.
+func GetIP(cluster *clusterv1.Cluster, machine *clusterv1.Machine) (string, error) {
```

Comment:
Where is this actually getting called, I thought the methods part of `Deployer` below are enough for the provider?

Reply:
I just did this as a convenience thing.